src/pages/FeelGoodPage.js

```diff
@@ -0,0 +1,55 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import pinkArrow from '../Images/arrow_pink.svg';
+
+export default function FeelGoodPage() {
+  const navigate = useNavigate();
+  return (
+    <Flex>
```

Comment:
This name doesn't really describes what the component is used for. you could rename it to 'FlexSection' 'PageWrapper', 'PageContainer' etc.